Makefile

```diff
@@ -22,9 +22,9 @@ install-python-requirements:
 install-python-requirements-dev: install-python-requirements
 	$(PYTHON) -m pip install -Ur requirements-dev.txt
 install-deps:
-	pkg install -q -y libucl py$(pyver)-ucl py$(pyver)-setuptools rsync python$(pyver) py$(pyver)-libzfs
+	pkg install -q -y libucl rsync git py$(pyver)-ucl py$(pyver)-libzfs
```

Comment:
I suggest we do a check for existing git executable first, and install the git-lite package if none was found.

Reply:
from previous build failures, that doesn't seem to have worked.

Reply:
```suggestion
	pkg install -q -y libucl rsync py$(pyver)-ucl py$(pyver)-libzfs
	if [ "`command -v git`" == "" ]; then pkg install -q -y git-list; fi 
```

Reply:
Never mind. If `git-lite` is not compatible, `git` should be installed just like you did.